enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Summarize.tsx

```diff
@@ -52,39 +52,20 @@ const SummarizeInner = ({
     onClose();
   };
 
-  const {
-    query,
-    stageIndex,
-    aggregations,
-    handleAddBreakout,
-    handleQueryChange,
-    handleRemoveBreakout,
-    handleReplaceBreakouts,
-    handleUpdateBreakout,
-    hasAggregations,
-  } = useSummarizeQuery({
-    query: currentQuery,
-    onQueryChange: setCurrentQuery,
-  });
+  const hasAggregations = Lib.aggregations(currentQuery, -1).length > 0;
```

Comment:
ignore this for now, I just needed it since I don't use `useSummarizeQuery` here. this will be split in the stacked PR for `Summarize`